docs/source/examples/loschmidt_echo_revival_zne.md

```diff
@@ -0,0 +1,396 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
```

Comment:
You can add tags to the tutorial. For an example, see https://github.com/unitaryfund/mitiq/blob/36e02854e6d92c75eccc0881b51e8ea9f18e8de2/docs/source/examples/calibration-tutorial.md?plain=1#L15

Reply:
Done in cbd9028.